.shellcheckrc

```diff
@@ -0,0 +1,2 @@
+# Disable file checks (otherwise every use of `gha-tools` will get flagged)
+disable=SC1091
```

Comment:
I think this should be added to the `ci-codeowners` block in CODEOWNERS:

https://github.com/rapidsai/raft/blob/23c942052bf17f8b116d7c6f79b37042296b93c3/.github/CODEOWNERS#L15-L17

and to the list of files that shouldn't trigger CI that needs GPUs:

https://github.com/rapidsai/raft/blob/23c942052bf17f8b116d7c6f79b37042296b93c3/.github/workflows/pr.yaml#L61-L82